website/docs/core-library/filters.md

```diff
@@ -14,9 +14,9 @@ All latitude/longitude coordinates are in decimal degrees. See the [H3 Index Rep
 
 | filter           | input     | outputs
 | ---------------- | --------- | -------
-| `geoToH3`        | lat/lon   | `H3Index`
-| `h3ToGeo`        | `H3Index` | cell center point in lat/lon
-| `h3ToGeoBoundary`| `H3Index` | cell boundary in lat/lon
+| `geoToH3`        | lat/lng   | `H3Index`
+| `h3ToGeo`        | `H3Index` | cell center point in lat/lng
+| `h3ToGeoBoundary`| `H3Index` | cell boundary in lat/lng
```

Comment:
Filters should be entirely renamed, as per the latest names of the binaries

Reply:
Updated!